src/tests/test_raft_repl_dev.cpp

```diff
@@ -152,6 +152,7 @@ TEST_F(RaftReplDevTest, Resync_From_Non_Originator) {
 }
 
 #if 0
```

Comment:
Can this test case be enabled or not yet?

Reply:
@Besroy had tried to enable this but failed.  The test of restart leader is passing in Storage hammer but not here..  @JacksonYao287 traiged this, pls share more findings?

Reply:
As far as I know, the UT's `write_on_leader` function cannot detect changes in the leader. If a leader switch occurs while writing data, this UT will get stuck. Although it works locally, it may fail occasionally in Jenkins, which is why it has been commented out again. @JacksonYao287 Please correct me if I'm wrong.